src/Illuminate/Mail/Mailer.php

```diff
@@ -325,7 +325,7 @@ protected function parseView($view)
     protected function addContent($message, $view, $plain, $raw, $data)
     {
         if (isset($view)) {
-            $message->setBody($this->renderView($view, $data), 'text/html');
+            $message->setBody($this->renderView($view, $data) ?: ' ', 'text/html');
```

Comment:
I wonder if the render view function should be updated to handle this, instead?

Reply:
No I don't think so. It's specific for the `setBody` that we need to do this.